### PR TITLE
Final cleaning

### PR DESCRIPTION
**Pull request by mghirsch42**

### CODE REVIEW

File: manifold/mnist_manifold.py [hunk 1]
```diff
@@ -18,17 +18,20 @@
...
 
 WIDTH = 28
 HEIGHT = 28
 EPSILON = 0.15  # example values: low = .01, high = .15
 PATH_TO_MODEL = "models/mnist_model.h5"
+MANIFOLD_FILE = "mnist-lle-60000-2d-2"
+MANIFOLD_SIZE = 60000  # The number of points in the manifold; used for visalization colors
 
 
 def convert_to_model(seq_model):
```
Comment (kteavery) [inline manifold/mnist_manifold.py]: do we need convert_to_model() in two places, or could we import?

**Review by kteavery — COMMENTED** [review]
I'm assuming your stuff more computationally intensive stuff compiles still. So apart from my convert_to_model() comment, I'm good.

File: manifold/mnist_manifold.py (hunk 1 above)
Comment (mghirsch42) [reply manifold/mnist_manifold.py]: Yes, the manifold stuff all still runs.
I separated out the convert_to_model() and the color list into mnist_tools. See what you think.

Comment (kteavery) [reply manifold/mnist_manifold.py]: looks good to me 👍

**Pull request merged**